docs/src/modeler_guide/time_series.md

```diff
@@ -96,6 +96,31 @@ it in memory. Here is an example of how to do this:
 sys = System(100.0; time_series_in_memory = true)
 ```
 
+### Compression
```

Comment:
It would be useful to cross link to the documentation in IS.

Reply:
Good point. I'll do it after there is a real link in the new docs.

Reply:
https://nrel-siip.github.io/InfrastructureSystems.jl/stable/dev_guide/time_series/